listenbrainz/webserver/static/js/src/fresh-releases/utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+
+export function formattedReleaseDate(releaseDate: string) {
```

Comment:
I'm more used to action verbs for functions that perform an action:
```suggestion
export function formatReleaseDate(releaseDate: string) {
```

Reply:
Good catch :D I like having properly worded function and variable names in the code too!